src/main/java/io/supertokens/thirdparty/ThirdParty.java

```diff
@@ -216,12 +216,11 @@ private static SignInUpResponse signInUpHelper(TenantIdentifierWithStorage tenan
                 AuthRecipeSQLStorage authRecipeStorage =
                         (AuthRecipeSQLStorage) tenantIdentifierWithStorage.getAuthRecipeStorage();
 
-                storage.startTransaction(con -> {
+                { // Try without transaction
```

Comment:
add more comment - why are we doing this?

Reply:
Done